website/docs/tooling/cli.md

```diff
@@ -81,8 +81,34 @@ function Comp(props) {
 
 ## Extraction
 
+<Tabs
+groupId="npm"
+defaultValue="npm"
+values={[
+{label: 'npm', value: 'npm'},
+{label: 'yarn', value: 'yarn'},
+]}>
+<TabItem value="npm">
+
+```sh
+npm formatjs extract --help
```

Comment:
I'm not sure this will work, do you mean to use `npx @formatjs/cli`?

Reply:
Sorry, my `yarn` is showing. I believe this should be `npm run`.